pkg/retry/retry.go

```diff
@@ -124,13 +127,13 @@ func newRetryTimerWithJitter(ctx context.Context, unit time.Duration, cap time.D
 
 // Default retry constants.
 const (
-	defaultRetryUnit = time.Second      // 1 second.
-	defaultRetryCap  = 30 * time.Second // 30 seconds.
+	defaultRetryUnit = 10 * time.Millisecond // 10 millisecond.
```

Comment:
Could you explain why this change ?

Reply:
Check the original values in dsync/retry.go

Reply:
Previous values were basically too long and too wide for a retry @vadmeste - they are only meant for clients but not for lockers. Lockers should retry faster.

I changed the value to more sustainable value i.e 50 Unit, 500 Cap